Fix/dqa

Camwyn opened the pull request:
[TEC-3737]

Addresses issues in [EXT-241]

- Alter "get tickets" to indicate sold-out events. Switch to "RSVP now" for RSVPs
- Fix ticket link alignment.
- Fix duped multiday events.
- Fix date alignment. 
- Fix ugly permalinks on activation.
- Fix styling with vertical filterbar.
- Fix Timezones!


https://d.pr/i/Cy1XEg

[EXT-241]: https://moderntribe.atlassian.net/browse/EXT-241
[TEC-3737]: https://moderntribe.atlassian.net/browse/TEC-3737

Camwyn committed:
Alter "get tickets" to indicate sold-out events.

Camwyn committed:
Fix ticket link alignment

Camwyn committed:
remove "on"

Camwyn committed:
Fix duped multiday events.

Camwyn committed:
Fix date alignment.
Fix sold out ticket display
Correct display for RSVP.

Camwyn committed:
Fix ugly permalinks on activation.

Camwyn committed:
Fix styling with vertical filterbar.

Camwyn committed:
Fix Timezones!

Camwyn committed:
Switch to "RSVP now"

bordoni commented on src/Tribe/Views/Summary_View.php:
```diff
@@ -122,20 +123,50 @@ protected function setup_template_vars() {
+
+		$is_multiday_start = false !== $event->multiday && $formatted_group_date === $formatted_start_date_beginning;
+		$is_multiday_end   = false !== $event->multiday && $formatted_group_date === $formatted_end_date_ending;
+		$is_all_day        = tribe_event_is_all_day( $event );
```
`$event->all_day`

Camwyn replied:
```suggestion
		$is_all_day        = $event->all_day;
```

Commit:
Update src/Tribe/Views/Summary_View.php

Camwyn committed:
Simplify template vars loop.
Set view to use the same list view limit to avoid conflicts and weird pagination.

Camwyn committed:
Merge branch 'fix/dqa' of github.com:mt-support/tribe-ext-summary-view into fix/dqa

Camwyn committed:
Remove commented-out code

bordoni commented on plugin.php:
```diff
@@ -90,4 +90,10 @@ function tribe_extension_summary_view_activation() {
+	// Fixes "ugly permalinks" on activation.
+	// Check seems silly, but might as well not do it if it's not needed.
+	if ( get_option('permalink_structure') ) {
+		flush_rewrite_rules();
```
`set_transient( '_tribe_events_delayed_flush_rewrite_rules', 'yes', 0 );`
Which will trigger a flush at the correct moment.

Camwyn replied:
Thanks, this was giving me a hard time!

bordoni commented on src/Tribe/Views/Summary_View.php:
```diff
@@ -122,20 +86,50 @@ protected function setup_template_vars() {
 
-		$formatted_start_date_beginning = tribe_beginning_of_day( $event->dates->start->format( Dates::DBDATEFORMAT ), tribe_get_option( 'dateWithoutYearFormat' ) );
-		$formatted_end_date_ending      = tribe_beginning_of_day( $event->dates->end->format( Dates::DBDATEFORMAT ), tribe_get_option( 'dateWithoutYearFormat' ) );
+		$format = tribe_get_option( 'dateWithoutYearFormat' );
```
`tribe_get_date_option( 'dateWithoutYearFormat', Tribe__Date_Utils::DBDATEFORMAT )`

bordoni commented on src/Tribe/Views/Summary_View.php:
```diff
@@ -122,20 +86,50 @@ protected function setup_template_vars() {
+			$is_all_day = true;
+		}
+
+		$end_date = $event->dates->end_display->format( get_option( 'time_format' ) );
```
Save `get_option( 'time_format' )` to a variable and re-use it.

Camwyn committed:
CR tweaks:
flush permalinks timing,
date and time options,
Add time zone to multiday events correctly.

Camwyn committed:
phpcs
